explanation/intro-to/high-availability.md

```diff
@@ -134,9 +130,9 @@ Packages in this list aren't necessarily **HA** related packages, but they have
 | tgt OR targetcli-fb* | [Ubuntu](https://launchpad.net/ubuntu/+source/tgt) \| [Upstream](https://github.com/fujita/tgt)
 | lvm2 | [Ubuntu](https://launchpad.net/ubuntu/+source/lvm2) \| [Upstream](https://sourceware.org/lvm2/)
 
-* **LVM2** in a Shared-Storage Cluster Scenario:
+* **LVM2** in a Shared-Storage Cluster Scenario:*
 <BR>**CLVM** - supported before **Ubuntu 20.04**
```

Comment:
Sphinx hasn't respected the new line, so we will probably need an empty line after this. The rendered docs have made all of this content into a single block.

I'd suggest getting rid of the html code `<br>` and just replacing it with proper spacing and line breaking :) it'll make the markdown more readable to do it that way.